### DISCUSSION TITLE
App language translation

### DISCUSSION THREAD

**Discussion by diamant3** [sumatrapdfreader/sumatrapdf#2551] [General]
Im translating the app language into my tagalog language but i dont know what to do about the word have "&", is this "and" or an address-of operator?

another question, there's a word "Man&ga Mode", how can i translate this? thank you

**Comment by GitHubRulesOK** [comment]
the & symbol is the prefix for the character to be used in the shortcut so in that case it would be in English Alt +V +<kbd> G </kbd> but the idea is not to duplicate another in the same menu area so Manga Mode is Right To Left (=RTL R2L/L2R)

Beware some menu items may not appear in some context so that possibly could have been M&anga but I am unsure if another view shortcut uses A Possibly there may be, but the M is used by Book&marks

![image](https://user-images.githubusercontent.com/12754913/167517396-ae8125a6-963a-4ffd-b3c5-2ccb4284858f.png)

**Reply by diamant3** [reply]
I got it, thanks for the explanation!
